recipe/bld.bat

```diff
@@ -60,7 +62,7 @@ cmake .. -LAH -G "NMake Makefiles"
     -DWITH_CUDA=0                                                                   ^
     -DWITH_OPENCL=0                                                                 ^
     -DWITH_OPENNI=0                                                                 ^
-    -DWITH_FFMPEG=0                                                                 ^
+    -DWITH_FFMPEG=1                                                                 ^
```

Comment:
This seems to be enabled, but `ffmpeg` is not added as a requirement for Windows. Am I missing something?